docs/source/pythonapi/examples/mgxs-part-i.ipynb

```diff
@@ -906,30 +908,32 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>0</th>\n",
-       "      <td>1</td>\n",
-       "      <td>(0.0e+00 - 6.3e-07)</td>\n",
-       "      <td>total</td>\n",
-       "      <td>(((total / flux) - (absorption / flux)) - (sca...</td>\n",
-       "      <td>4.884981e-15</td>\n",
-       "      <td>0.011274</td>\n",
+       "      <td> 1</td>\n",
+       "      <td> 0.000000</td>\n",
+       "      <td>  0.000001</td>\n",
+       "      <td> total</td>\n",
+       "      <td> (((total / flux) - (absorption / flux)) - (sca...</td>\n",
+       "      <td> 4.884981e-15</td>\n",
+       "      <td> 0.011274</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>1</th>\n",
-       "      <td>1</td>\n",
-       "      <td>(6.3e-07 - 2.0e+01)</td>\n",
-       "      <td>total</td>\n",
-       "      <td>(((total / flux) - (absorption / flux)) - (sca...</td>\n",
-       "      <td>1.221245e-15</td>\n",
-       "      <td>0.001802</td>\n",
+       "      <td> 1</td>\n",
+       "      <td> 0.000001</td>\n",
+       "      <td> 20.000000</td>\n",
+       "      <td> total</td>\n",
+       "      <td> (((total / flux) - (absorption / flux)) - (sca...</td>\n",
+       "      <td> 1.221245e-15</td>\n",
+       "      <td> 0.001802</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "   cell         energy [MeV] nuclide  \\\n",
-       "0     1  (0.0e+00 - 6.3e-07)   total   \n",
-       "1     1  (6.3e-07 - 2.0e+01)   total   \n",
+       "   cell  energy low [MeV]  energy high [MeV] nuclide  \\\n",
+       "0     1          0.000000           0.000001   total   \n",
```

Comment:
Personally I think the scientific notation is more informative here. Can you make Pandas print it out using scientific notation instead?